api/types.go

```diff
@@ -349,7 +349,7 @@ type ForkUpgradeParams struct {
 	UpgradeLightningHeight   abi.ChainEpoch
 	UpgradeThunderHeight     abi.ChainEpoch
 	UpgradeWatermelonHeight  abi.ChainEpoch
-	UpgradePineappleHeight   abi.ChainEpoch
+	UpgradeDragonHeight      abi.ChainEpoch
```

Comment:
Mango as well?

Reply:
Or are we saying that this hight isn't configurable?

Reply:
We should still have it, it's not so much about configurability as information about the network.